code/modules/mob/living/carbon/human/species_types.dm

```diff
@@ -474,26 +475,32 @@ var/global/image/plasmaman_on_fire = image("icon"='icons/mob/OnFire.dmi', "icon_
 	safe_toxins_max = 0
 	dangerous_existence = 1 //So so much
 	need_nutrition = 0 //Hard to eat through a helmet
+	burnmod = 2
+	heatmod = 2
+	speedmod = 1
 	var/skin = 0
 
 /datum/species/plasmaman/skin
 	name = "Skinbone"
 	skin = 1
+	roundstart = 0
 
 /datum/species/plasmaman/update_base_icon_state(mob/living/carbon/human/H)
 	var/base = ..()
-	if(base == id)
-		base = "[base][skin]"
+	if(base == id && !skin)
+		base = "[base]"
+	else
+		base = "skinbone_s"
```

Comment:
If I do that it breaks the previews. It's a lot of fucking bullshit, but whatever. Theoretically i should actually change the skinbone sprite to skinbone_m_s for that one to work as well